__testfixtures__/fourteen-testing-api/basic-application-test.output.js

```diff
@@ -0,0 +1,13 @@
+import { afterEach, beforeEach, describe, it } from 'mocha';
+import { setupApplicationTest } from 'ember-mocha';
+import { expect } from 'chai';
+
+describe('basic acceptance test', function() {
+  setupApplicationTest();
+
+  it('can visit /', async function() {
+    await visit('/');
```

Comment:
would this need to add the `visit` import too?

Reply:
It seems `ember-qunit-codemod` does the same [here](https://github.com/rwjblue/ember-qunit-codemod/blob/1b123a78e94bdd88008693ebb69d797be86a877c/__testfixtures__/ember-qunit-codemod/module-for-acceptance.output.js). I.e. they just transform the global helpers to use `await`, but still using the old ones, not importing the new ones from `@ember/test-helpers`. Which seems ok, given that `ember-test-helpers-codemod` does the conversion you are looking for?

Reply:
@simonihmig @Turbo87 Yeah, this is correct.  The assumption is that this would be done in congress with `ember-test-helpers-codemod`.